.circleci/config.yml

```diff
@@ -34,32 +31,31 @@ jobs:
       - setup_remote_docker
       - run:
           name: Run Integration Test
-          working_directory: /go
           command: |
             # Spin up environment
-            ACCEPT_EULA=yes docker-compose -f src/github.com/qlik-oss/corectl/test/docker-compose.yml up -d
+            ACCEPT_EULA=yes docker-compose -f ./test/docker-compose.yml up -d
             ENGINE_1_CONTAINER_ID=$(docker ps -aqf "name=engine-1")
             TEST_HOST=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.Gateway}}{{end}}' "$ENGINE_1_CONTAINER_ID")
             # Start a container for test execution
-            TEST_CONTAINER=$(docker run -e TEST_HOST=$TEST_HOST -d golang:1.9-alpine tail -f /dev/null)
+            TEST_CONTAINER=$(docker run -e TEST_HOST=$TEST_HOST -d golang:1.11-alpine tail -f /dev/null)
             # Install packages
-            docker exec $TEST_CONTAINER sh -c 'apk update && apk add --no-cache bash'
+            docker exec $TEST_CONTAINER sh -c 'apk update && apk add --no-cache bash gcc git libc-dev'
             # Copy code into container
-            docker cp . $TEST_CONTAINER:/go
+            docker cp /go/pkg $TEST_CONTAINER:/go/pkg
+            docker cp . $TEST_CONTAINER:/corectl
             # Copy apps and data into the first engine container, the second one does not need it.
-            chmod -R 777 src/github.com/qlik-oss/corectl/test/
-            docker cp src/github.com/qlik-oss/corectl/test/apps/ $ENGINE_1_CONTAINER_ID:/
-            docker cp src/github.com/qlik-oss/corectl/test/data/ $ENGINE_1_CONTAINER_ID:/
+            chmod -R 777 ./test/
+            docker cp ./test/apps/ $ENGINE_1_CONTAINER_ID:/
+            docker cp ./test/data/ $ENGINE_1_CONTAINER_ID:/
             # Execute tests
-            docker exec $TEST_CONTAINER /bin/bash -c 'go test /go/src/github.com/qlik-oss/corectl/test/corectl_integration_test.go --engineIP $TEST_HOST:9076 --engine2IP $TEST_HOST:9176'
+            docker exec $TEST_CONTAINER /bin/bash -c 'cd /corectl && go test /corectl/test/corectl_integration_test.go --engineIP $TEST_HOST:9076 --engine2IP $TEST_HOST:9176'
```

Comment:
Need to execute `go` commands outside of the `GOPATH` to use the modules support